libfuse/fs.go

```diff
@@ -199,18 +199,22 @@ func (f *FS) Root() (fs.Node, error) {
 
 // Statfs implements the fs.FSStatfser interface for FS.
 func (f *FS) Statfs(ctx context.Context, req *fuse.StatfsRequest, resp *fuse.StatfsResponse) error {
-	// TODO: Fill in real values for these.
-	var bsize uint32 = 32 * 1024
 	*resp = fuse.StatfsResponse{
-		Blocks:  ^uint64(0) / uint64(bsize),
-		Bfree:   ^uint64(0) / uint64(bsize),
-		Bavail:  ^uint64(0) / uint64(bsize),
-		Files:   0,
-		Ffree:   0,
-		Bsize:   bsize,
+		Bsize:   fuseBlockSize,
 		Namelen: ^uint32(0),
-		Frsize:  0,
+		Frsize:  fuseBlockSize,
 	}
+	status, _, err := f.config.KBFSOps().Status(ctx)
```

Comment:
Ah, good point on Finder might show total free space. 

> whenever there's a merged MD update on any TLF we have an FBO for
Doesn't this suffer from the same problem where we might not be tracking all TLFs updates? Or did you mean this is still better than pure time-based cache busting?

What expiration time do you think is a good choice? I'm thinking more of a 5s rather than 1m